commit 330887a7543920f463f75e55d4577a6ab8c5d957 (refs/changes/70/2870/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-10-25T14:21:11-07:00 (1 year, 11 months ago)
    
    TOOLS-1898 sdcnode updates for Node.js Oct 2017 vulns
    Reviewed by: Trent Mick <trentm@gmail.com>
    Approved by: Trent Mick <trentm@gmail.com>

diff --git a/Makefile b/Makefile
index e5dacdfa..6569c29e 100644
--- a/Makefile
+++ b/Makefile
@@ -36,7 +36,7 @@ JSSTYLE_FLAGS	 = -o "indent=2,doxygen,unparenthesized-return=0,line-length=120"
 REPO_MODULES	 =
 SMF_MANIFESTS_IN = smf/manifests/adminui.xml.in
 
-NODE_PREBUILT_VERSION=v4.8.4
+NODE_PREBUILT_VERSION=v4.8.5
 ifeq ($(shell uname -s),SunOS)
 	NODE_PREBUILT_IMAGE=18b094b0-eb01-11e5-80c1-175dac7ddf02
 	NODE_PREBUILT_TAG=zone
